build_tools/llvm_version.txt

```diff
@@ -1 +1 @@
-0078cf79adc2f24a168bc774cba1f39dda5e3752
+f427890a1dcd7759240568f189df8fbbcf23669c
```

Comment:
Can you modify commit to c60f24dca96dad44afc60df3fcb80997737b6306

Reply:
Alternative is just to wait and integrate this change once we're able to in OpenXLA

Reply:
Yes it should have that commit since that landed 2 weeks ago.

Reply:
done